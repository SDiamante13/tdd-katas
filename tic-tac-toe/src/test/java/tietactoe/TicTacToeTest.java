package tietactoe;

import org.approvaltests.Approvals;
import org.approvaltests.StoryBoard;
import org.approvaltests.reporters.QuietReporter;
import org.approvaltests.reporters.UseReporter;
import org.junit.jupiter.api.Test;

@UseReporter(QuietReporter.class)
class TicTacToeTest {

    @Test
    void ticTacToeGame() {
        StoryBoard storyBoard = new StoryBoard();
        storyBoard.addDescription("Tic-Tac-Toe");

        TicTacToe ticTacToe = new TicTacToe();
        storyBoard.addFrame("New Game", ticTacToe);

        Approvals.verify(storyBoard);
    }
}
