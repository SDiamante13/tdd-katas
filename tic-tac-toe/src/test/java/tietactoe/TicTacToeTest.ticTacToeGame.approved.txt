Tic-Tac-Toe


New Game:
tietactoe.TicTacToe@4923ab24